pkg/kubectl/cmd/util/factory.go

```diff
@@ -206,6 +206,7 @@ const (
 	SecretForDockerRegistryV1GeneratorName      = "secret-for-docker-registry/v1"
 	SecretForTLSV1GeneratorName                 = "secret-for-tls/v1"
 	ConfigMapV1GeneratorName                    = "configmap/v1"
+	ClusterRoleBindingV1GeneratorName           = "clusterrolebinding.rbac.authorization.k8s.io/v1alpha1"
```

Comment:
I'm a little bit hesitant to use this lengthy generator name. Although this one will not be probably exposed to users.